### PR TITLE
Disabled accessibility tests for clinical-lowlight-theme

### PR DESCRIPTION
**Pull request by avinashg1994**
### Summary

Added a check to disable accessibility tests for **clinical-lowlight-theme** to prevent the color contrast violations.

#351

### CODE REVIEW

File: src/wdio/services/TerraCommands/accessibility.js [hunk 1]
```diff
@@ -10,8 +10,11 @@ import determineOptions from './determine-test-options';
  * @param {Object[]} [options.viewports] - the list of Terra viewports to test.
  */
 const runAccessibilityTest = (options) => {
-  const axeResults = global.browser.axe(options);
-  global.expect(axeResults).to.be.accessible();
+  const { theme } = global.browser.options;
+  if (theme !== 'clinical-lowlight-theme') {
```
Comment (nramamurth) [inline src/wdio/services/TerraCommands/accessibility.js]: Is this for short-term or long-term? In either case, I think it is better to have a follow-up issue to remove this check whenever a decision is made to test accessibility in lowlight.

**Comment by nramamurth** [comment]
@neilpfeiffer, I do not seem to recollect the actual reason — why exactly do we have color contrast issues in Lowlight? Will we go back and _fix_ them in the future?

**Review by emilyrohrbough — CHANGES_REQUESTED** [review]
Please see the comment left on the issue:
https://github.com/cerner/terra-toolkit/issues/351#issuecomment-553474034

File: src/wdio/services/TerraCommands/determine-test-options.js [hunk 2]
```diff
@@ -49,11 +49,12 @@ const axeOptions = (args) => {
     options, viewports,
   } = determineArgs(args);
 
+  const globalAxeRules = global.browser.options.axe.options.rules;
   const axeRules = options.rules || options.axeRules;
 
   return {
     ...viewports && { viewports },
-    ...axeRules && { rules: axeRules },
+    ...axeRules && { rules: { ...globalAxeRules, ...axeRules } },
```
Comment (emilyrohrbough) [inline src/wdio/services/TerraCommands/determine-test-options.js]: I'm not sure this change is needed. Did the axe tool not properly configure with the root level configuration? Do you have a couple test runs as evidence you can link?

**Pull request closed without merging**